js/.jscsrc

```diff
@@ -28,7 +28,7 @@
   "requireSpaceAfterLineComment": true,
   "requireSpaceBeforeBinaryOperators": ["+", "-", "/", "*", "=", "==", "===", "!=", "!==", ">", "<", ">=", "<="],
   "requireSpaceBetweenArguments": true,
-  "requireSpacesInAnonymousFunctionExpression": { "beforeOpeningCurlyBrace": true, "beforeOpeningRoundBrace": true },
```

Comment:
It just fails but I'm not sure if it's a regression or intentional.
On Sep 2, 2015 10:09, "Chris Rebert" <notifications@github.com> wrote:

> In js/.jscsrc
> <https://github.com/twbs/bootstrap/pull/17436#discussion_r38503896>:
>
> > @@ -28,7 +28,7 @@
> >    "requireSpaceAfterLineComment": true,
> >    "requireSpaceBeforeBinaryOperators": ["+", "-", "/", "*", "=", "==", "===", "!=", "!==", ">", "<", ">=", "<="],
> >    "requireSpaceBetweenArguments": true,
> > -  "requireSpacesInAnonymousFunctionExpression": { "beforeOpeningCurlyBrace": true, "beforeOpeningRoundBrace": true },
>
> Was the rule removed from JSCS itself, or do we not currently comply with
> it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twbs/bootstrap/pull/17436/files#r38503896>.
>